src/rpdk/core/data/schema/provider.definition.schema.v1.json

```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "https://json-schema.org/draft-07/schema#",
-    "$id": "provider.definition.schema.v1.json",
+    "$id": "https://schema.cloudformation.us-east-1.amazonaws.com/provider.definition.schema.v1.json",
```

Comment:
Is this link valid? Getting access denied when I try to view. Might not be related to this PR, but we should really depend on a central source for the schema just so we don't have to keep copying it over whenever there's changes.

Reply:
Ugh, yep if it's in the schema it should: https://github.com/aws-cloudformation/aws-cloudformation-resource-schema/blob/1fd31cdb3ca76f8418c0209753e204e4e12c019a/src/main/resources/schema/provider.definition.schema.v1.json#L3